packages/airnode-node/src/evm/handlers/fetch-pending-requests.test.ts

```diff
@@ -18,19 +18,20 @@ describe('fetchPendingRequests', () => {
           airnodeAddress: '0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace',
           chainId: '31337',
           requesterAddress: '0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512',
-          sponsorWalletAddress: '0x654B6d01E1A03eeF91F50D79203Ace648be81350',
+          sponsorWalletAddress: '0x91Fa5bf7FE3cF2a8970B031b1EB6f824fFe228BE',
           encodedParameters:
             '0x316262626262000000000000000000000000000000000000000000000000000066726f6d000000000000000000000000000000000000000000000000000000004554480000000000000000000000000000000000000000000000000000000000746f00000000000000000000000000000000000000000000000000000000000055534400000000000000000000000000000000000000000000000000000000005f74797065000000000000000000000000000000000000000000000000000000696e7432353600000000000000000000000000000000000000000000000000005f70617468000000000000000000000000000000000000000000000000000000726573756c7400000000000000000000000000000000000000000000000000005f74696d657300000000000000000000000000000000000000000000000000003130303030300000000000000000000000000000000000000000000000000000',
           endpointId: '0x13dea3311fe0d6b84f4daeab831befbc49e19e6494c41e9e065a09c3c68f43b6',
           fulfillAddress: '0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512',
           fulfillFunctionId: '0x7c1de7e1',
-          id: '0xe6c05b0ddd2f48a6b0f6ea52f17e465aefa02eaa4c3402509f22d40e092d042f',
+          id: '0x8d6d6ce1ac5020c4db26c34ae42fa8318a51fa21faef9a65663724641b369e63',
           metadata: {
             address: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
-            blockNumber: 12,
+            blockNumber: 13,
             currentBlock: null,
             ignoreBlockedRequestsAfterBlocks: 20,
-            transactionHash: '0x0ba1b3c4b6710fffeb6c2d1e62f11fa695177a456e4e895f0666ec54b2e7f8bf',
+            transactionHash:
+              '0xba69a4584ca8e383412741f9f75b351d34d5eff9e14b38d8ba8b994bb175e3c5549594040952fc82817cbfb3eaf4593aa8858a34b682886737aac15f759dc1d41c',
```

Comment:
Why is it that this transaction hash is much longer than other transaction hashes?

Reply:
It's probably a bug - at one place I incorrectly renamed one hash and then I reverted that for failing tests. This is probably never used anywhere so the tests pass without it.